### PR TITLE
Remove code for deploying on PaaS

### PR DESCRIPTION
**Pull request by klssmith**
Now that the app is running on ECS, we can remove everything that was specific to running on the PaaS.

This will fail until https://github.com/alphagov/notifications-aws/pull/1985 is merged.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,35 +28,15 @@ generate-manifest:
```
Comment (idavidmcdonald) [inline Makefile:28]: generate-manifest can be deleted too

File: Makefile [hunk 2]
```diff
@@ -36,35 +28,15 @@ generate-manifest:
...
 
 .PHONY: preview
 preview: ## Set environment to preview
-	$(eval export CF_SPACE=preview)
-	$(eval export ROUTE='docs.notify.works')
 	$(eval export HOST='https://docs.notify.works')
 	@true
 
 .PHONY: production
 production: ## Set environment to production
```
Comment (idavidmcdonald) [inline Makefile:40]: One to double check, but do we still need the environment ones like `make preview` and `make production`

File: Makefile (hunk 1 above)
Comment (klssmith) [reply Makefile]: Good spot, I've now removed that too

File: Makefile (hunk 2 above)
Comment (klssmith) [reply Makefile:35]: Yes, we still use those - they're used in the Dockerfile to generate the yml for the right environment

**Pull request merged**